contrib/storage.conf

```diff
@@ -0,0 +1,28 @@
+# storage.conf is the configuration file for all tools
+# that share the containers/storage libraries
+# See man 5 containers-storage.conf for more information
```

Comment:
Not sure, should this be 'man 5 containers-storage.conf" or "containers-storage.conf(5)"?  I think the second.

Reply:
the former works just fine too.